lib/chef/resource/windows_certificate.rb

```diff
@@ -92,7 +121,9 @@ class WindowsCertificate < Chef::Resource
         default: false,
         introduced: "16.8"
 
-      action :create, description: "Creates or updates a certificate" do
+      action :create do
```

Comment:
This needs to be the way it was before for the documentation to work

Reply:
Weird, I didn't change that, I thought someone else had.. will put that back